hu/admin/server.php

```diff
@@ -9,26 +9,26 @@
-        'build_updated' => 'The build details for this server have been updated. Some changes may require a restart to take effect.',
-        'suspension_toggled' => 'Server suspension status has been changed to :status.',
-        'rebuild_on_boot' => 'This server has been marked as requiring a Docker Container rebuild. This will happen the next time the server is started.',
-        'install_toggled' => 'The installation status for this server has been toggled.',
-        'server_reinstalled' => 'This server has been queued for a reinstallation beginning now.',
-        'details_updated' => 'Server details have been successfully updated.',
-        'docker_image_updated' => 'Successfully changed the default Docker image to use for this server. A reboot is required to apply this change.',
-        'node_required' => 'You must have at least one node configured before you can add a server to this panel.',
-        'transfer_nodes_required' => 'You must have at least two nodes configured before you can transfer servers.',
-        'transfer_started' => 'Server transfer has been started.',
-        'transfer_not_viable' => 'The node you selected is not viable for this transfer.',
+        'startup_changed' => 'A szerver indítási konfigurációi frissültek. Ha ennek a szervernek a Nest-jét vagy a Egg-ét megváltoztatta, akkor újratelepítés fog bekövetkezni most.',
+        'server_deleted' => 'A szerver sikeresen törlésre került a rendszerből.',
+        'server_created' => 'A szerver sikeresen létrehozva a penelben. Kérlek adj pár percet a daemonnak hogy befejezze a szerver telepítését.',
+        'build_updated' => 'A szerver build paraméterei frissültek. Bizonyos változások életbe lépéséhez újraindítás szükséges.',
+        'suspension_toggled' => 'A szerver felfüggesztési állapota megváltozott :status.',
+        'rebuild_on_boot' => 'Ez a szerver Docker Container újraépítésre lett jelölve. Ez a következő szerverindításnál fog bekövetkezni.',
+        'install_toggled' => 'A szerver telepítési állapota bekapcsolva.',
+        'server_reinstalled' => 'Ez a szerver sorba került újratelepítésre ami most veszi kezdetét.',
+        'details_updated' => 'A szerver részletei sikeresen frissültek.',
+        'docker_image_updated' => 'Successfully changed the default Docker image to use for this server. A reboot is required to apply this change. Sikeresen megváltoztatta a szerverhez használt alapértelmezett Docker-képet. A változások életbe lépéséhez újraindítás szükséges.',
+        'node_required' => 'Legalább egy, már konfigurált csomópontra van szükség hogy szervert hozhass létre a panelen belül.',
+        'transfer_nodes_required' => 'Legalább kettő, már konfigurált csomópontra van szükséged hogy szervert migrálhass.',
+        'transfer_started' => 'A szerver migrálása elkezdődött.',
+        'transfer_not_viable' => 'A kiválasztott csomópont nem alkalmas erre a migrálásra.',
```

Comment:
Itt a migrálás helyett inkább áthelyezés?

Reply:
Máshol is akkor átrakom